Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml

```diff
@@ -578,6 +578,9 @@
     flavors:
       - meaty
       - spicy
+  - type: Tag
+    tags:
+    - Meat
```

Comment:
Usually components are sorted by how important they are, with things like the `SpriteComponent` an top of the list. Could you move the tag below the `SolutionContainerManager`?

Reply:
like so?